README.md

```diff
@@ -174,6 +174,7 @@
 ### PureScript
 
 * [Easy PureScript Nix](https://github.com/justinwoo/easy-purescript-nix) - A project to easily use PureScript and other tools with Nix.
+* [purs-nix](https://github.com/purs-nix/purs-nix) - Manage your PureScript projects with Nix.
```

Comment:
Description should probably indicate its a combined CLI and library, but its otherwise accurate